polaris/ocean/tasks/cosine_bell/analysis.py

```diff
@@ -47,143 +31,73 @@ def __init__(self, component, resolutions, icosahedral, subdir,
+            For the default method, we always use the initial state.
+
+        zidx : int, optional
+            The z-index for the vertical level at which to evaluate the exact
+            solution
 
         Returns
         -------
-        rmseValue : float
-            The root-mean-squared error
-
-        nCells : int
-            The number of cells in the mesh
+        solution: np.ndarray of type float
+            The exact solution
         """
 
+        if field_name != 'tracer1':
+            print(f'Variable {field_name} not available as an analytic '
+                  'solution for the cosine_bell test case')
+
         config = self.config
         latCent = config.getfloat('cosine_bell', 'lat_center')
         lonCent = config.getfloat('cosine_bell', 'lon_center')
         radius = config.getfloat('cosine_bell', 'radius')
         psi0 = config.getfloat('cosine_bell', 'psi0')
-        convergence_eval_time = config.getfloat('spherical_convergence',
-                                                'convergence_eval_time')
+        vel_pd = config.getfloat('cosine_bell', 'vel_pd')
 
         ds_mesh = xr.open_dataset(f'{mesh_name}_mesh.nc')
         ds_init = xr.open_dataset(f'{mesh_name}_init.nc')
-        # find time since the beginning of run
-        ds = xr.open_dataset(f'{mesh_name}_output.nc')
-        for j in range(len(ds.xtime)):
-            tt = str(ds.xtime[j].values)
-            tt.rfind('_')
-            DY = float(tt[10:12]) - 1
-            if DY == convergence_eval_time:
-                sliceTime = j
-                break
-        HR = float(tt[13:15])
-        MN = float(tt[16:18])
-        t = 86400.0 * DY + HR * 3600. + MN
         # find new location of blob center
         # center is based on equatorial velocity
         R = ds_mesh.sphere_radius
-        distTrav = 2.0 * 3.14159265 * R / (86400.0 * convergence_eval_time) * t
         # distance in radians is
-        distRad = distTrav / R
+        distRad = 2.0 * np.pi * time / (86400.0 * vel_pd)
         newLon = lonCent + distRad
         if newLon > 2.0 * np.pi:
             newLon -= 2.0 * np.pi
-
-        # construct analytic tracer
         tracer = np.zeros_like(ds_init.tracer1[0, :, 0].values)
-        latC = ds_mesh.latCell.values
-        lonC = ds_mesh.lonCell.values
+        latC = ds_init.latCell.values
+        lonC = ds_init.lonCell.values
+        # TODO replace this with great circle distance
         temp = R * np.arccos(np.sin(latCent) * np.sin(latC) +
                              np.cos(latCent) * np.cos(latC) * np.cos(
             lonC - newLon))
         mask = temp < radius
         tracer[mask] = (psi0 / 2.0 *
                         (1.0 + np.cos(3.1415926 * temp[mask] / radius)))
-
-        # oad forward mode data
-        tracerF = ds.tracer1[sliceTime, :, 0].values
-        rmseValue = np.sqrt(np.mean((tracerF - tracer)**2))
-
-        return rmseValue, ds_init.dims['nCells']
+        return tracer
```

Comment:
This can be an `xarray.DataArray` but we may want to make that change in #127 rather than here.
```suggestion
        return xr.DataArray(data=tracer, dims=('nCells',))
```